dojo/api_v2/serializers.py

```diff
@@ -1312,9 +1359,7 @@ def save(self, push_to_jira=False):
             scan_date_time = timezone.make_aware(scan_date_time, timezone.get_default_timezone())
         verified = data['verified']
         active = data['active']
-        version = None
-        if 'version' in data:
-            version = data['version']
+        version = data.get('version', None)
```

Comment:
None is the default value when the key is not present so:
```suggestion
        version = data.get('version')
```

Reply:
not very useful to comment on draft PRs with old code that is already changed elsewhere.